### ISSUE TITLE
Readme.md or CONTRIBUTING.md 

### ISSUE THREAD

**Issue by disconsented** [UnicornHeartClub/Issues#140]
Adding a Readme.md would help direct people who are unfamiliar with GitHub towards the issues section.

Contributing.md would hopefully cut down on the eventually duplicate issues.

**Comment by tlackemann** [comment]
Agreed! This is actually supposed to be "temporary" but each day it grows more and more permanent. Initially it was just so duplicate bugs wouldn't be reported as the forums begin to grow as well.

I'm open to additional bug trackers or thoughts, if not a README with some contributing info I agree would go a long way.

**Comment by Aquaricat** [comment]
Personally, I like the idea of a single centralized tracker that's used across the projects, especially since portions of PVTT are decidedly **not** open source.  A lot of the management to keep tickets sorted can be done via labels, which @Pjb518 and I have been being careful to do.  It keeps everything together, and we can easily reference related issues back and forth when a bug crosses borders from one feature to another.

On the other hand, there's an equally strong case for using the individual project trackers.  Using the individual trackers means that the reports are limited to that project, and can be properly referenced in project commits and pull requests.  It keeps things more focused, and pushes toward a more detailed and complete development history than the central tracker does.
